app/views/comfy/admin/cms/fragments/_form_fragment_attachments.html.haml

```diff
@@ -4,9 +4,19 @@
       - variant = attachment.variant(Comfy::Cms::File::VARIANT_SIZE[:thumb])
       - thumb = image_tag(url_for(variant), size: "200x150")
     .fragment-attachment.btn-group.btn-group-sm.mb-1
-      - filename = attachment.filename.to_s
-      - truncated_filename = truncate(filename, length: 40, omission: "...#{filename.last(10)}")
-      = link_to truncated_filename, attachment, data: {toggle: "page-file-popover", content: thumb}, class: "btn btn-light text-truncate", target: "_blank"
+      :ruby
+        filename = attachment.filename.to_s
+        truncated_filename = truncate(filename, length: 40, omission: "...#{filename.last(10)}")
+        cms_page_file_link_tag = [
+          "{{ cms:page_file_link #{fragment_id}",
+          (", filename: \"#{filename}\"" if multiple),
+          (", as: image" if attachment.image?),
+          " }}".html_safe
+        ].compact.join
```

Comment:
Kinda ugly. Let's move that into admin view helper.

Reply:
Done